modules/tensor_mechanics/test/tests/uexternaldb/tests

```diff
@@ -68,7 +68,7 @@
     library_mode = 'DYNAMIC'
     valgrind = 'NONE'
     cli_args = 'UserObjects/uexternaldb/execute_on=INITIAL'
-    expect_out = 'getoutdir           1 \. '
+    expect_out = 'getoutdir           0 '
```

Comment:
I don't want to panic anyone, but while I was verifying that this was the same behavior change as in the test below, I discovered *Fortran code in MOOSE*...  :scream:

Reply:
Can I ... resolve this? Run a git blame on the Fortran stuff...

Reply:
Ha; sorry.  There really needs to be a "conversation that doesn't need to be resolved" option on Github.  My creativity is just plain squelched if I'm disincentivized from making stupid wisecracks while I work, but some wisecracks are best associated with particular line numbers.